.github/workflows/ci.yml

```
@@ -21,5 +21,6 @@ jobs:
         run: uv sync 
 
       - name: Run pre-commit checks
-        run: |
+        run: 
```

yamllint

[error] trailing spaces

(trailing-spaces)